From 8f8463bff2adbc687d0614fc51f173c8c7290a5c Mon Sep 17 00:00:00 2001
From: Liu Shixin <liushixin2@huawei.com>
Date: Wed, 13 Jul 2022 11:09:46 +0800
Subject: [PATCH 3575/4585] [Huawei] Revert "arm64: mm: hugetlb: add support
 for free vmemmap pages of HugeTLB"

Offering: HULK
hulk inclusion
category: feature
bugzilla: 187198

--------------------------------

There is a formal solution to support hugetlb vmemmap feature on arm64.

This reverts commit 5838d235a395e221968e1f024bfaf8650278a522.

Signed-off-by: Liu Shixin <liushixin2@huawei.com>
---
 fs/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/Kconfig b/fs/Kconfig
index 6e723c90a506..37a8895a3254 100644
--- a/fs/Kconfig
+++ b/fs/Kconfig
@@ -239,7 +239,7 @@ config HUGETLB_PAGE
 
 config HUGETLB_PAGE_FREE_VMEMMAP
 	def_bool HUGETLB_PAGE
-	depends on X86_64 || ARM64
+	depends on X86_64
 	depends on SPARSEMEM_VMEMMAP
 
 config HUGETLB_PAGE_FREE_VMEMMAP_DEFAULT_ON
-- 
2.17.1

